systemtests/utils/utils.go

```diff
@@ -53,14 +54,14 @@ func StopOnError(testFailed bool) {
 }
 
 // ConfigCleanupCommon performs common cleanup after each test
-func ConfigCleanupCommon(t *testing.T, nodes []TestbedNode) {
+func ConfigCleanupCommon(t *testing.T, nodes []stu.TestbedNode) {
 
 	if !OkToCleanup(t.Failed()) {
 		return
 	}
 
 	for _, node := range nodes {
-		cmdStr := "sh -c 'sudo $GOSRC/github.com/contiv/netplugin/scripts/cleanup'"
```

Comment:
I vaguely remember these changes were needed when we brought in  DIND tests. I think it is ok to revert these now since we don't run DIND sanity anymore, but just making a note of it for myself in case we need to revisit DIND sanity in future.